### PR TITLE
Replace all usages of GET /context-api/v2/projects/{projectId}/match/{matchId} to the new Context API endpoint: GET /context-api/v2/projects/{projectId}/processes/{processUid}

### PR DESCRIPTION
**Pull request by vbarhatov**

### CODE REVIEW

File: visual-context/python/html-match-all/html_match_all.py [hunk 1]
```diff
@@ -70,32 +70,32 @@ def main():
...
-        print('Waiting for matching to complete; status = ' + match_status)
+    while process_status != 'COMPLETED':
+        print('Waiting for matching to complete; status = ' + process_status)
         time.sleep(5)
         resp = requests.get(url, headers = headers)
         if resp.status_code != 200:
             print(resp.status_code)
             print(resp.text)
             sys.exit()
-        match_status = resp.json()['response']['data']['status']
```
Comment (MihailTymoshenko) [inline visual-context/python/html-match-all/html_match_all.py:96]: _processState_ here also

File: visual-context/python/html-match-fileuri/html_match_fileuri.py [hunk 2]
```diff
@@ -74,32 +74,32 @@ def main():
...
-        print('Waiting for matching to complete; status = ' + match_status)
+    while process_status != 'COMPLETED':
+        print('Waiting for matching to complete; status = ' + process_status)
         time.sleep(5)
         resp = requests.get(url, headers = headers)
         if resp.status_code != 200:
             print(resp.status_code)
             print(resp.text)
             sys.exit()
-        match_status = resp.json()['response']['data']['status']
```
Comment (MihailTymoshenko) [inline visual-context/python/html-match-fileuri/html_match_fileuri.py:100]: _processState_

**Review by MihailTymoshenko — CHANGES_REQUESTED** [review]

File: visual-context/python/html-match-all/html_match_all.py (hunk 1 above)
Comment (vbarhatov) [reply visual-context/python/html-match-all/html_match_all.py:96]: fixed

File: visual-context/python/html-match-fileuri/html_match_fileuri.py (hunk 2 above)
Comment (vbarhatov) [reply visual-context/python/html-match-fileuri/html_match_fileuri.py:100]: fixed

**Pull request merged**